### ISSUE TITLE
Variables don't display for queries

### ISSUE THREAD

**Issue by afrankel-sfdo** [apollographql/apollo-client-devtools#539]
**Intended outcome:**

Any variables executed in a query should show in the active queries inspector. Mutations are fine.

**Actual outcome:**

Variables are always empty for queries.

**How to reproduce the issue:**

### Network tab
```
{"operationName":"brandSettings","variables":{"id":123123},"query":"query brandSettings($id: Int!) {\n  organization(id: $id) {\n    id\n    orgSettings {\n      defaultLocale\n      __typename\n    }\n    brandSettings {\n      communityLogo\n      footerContent1\n      footerContent2\n      footerContent3\n      primaryColor\n      secondaryColor\n      localized {\n        language\n        communityLogo\n        footerContent1\n        footerContent2\n        footerContent3\n        __typename\n      }\n      __typename\n    }\n    __typename\n  }\n}\n"}
```

###  Apollo dev tools
![Screen Shot 2021-04-26 at 9 26 44 AM](https://user-images.githubusercontent.com/31106469/116109725-4b9d1380-a672-11eb-8659-585b7ec44919.png)

### Issue
network request shows `variables":{"id":123123}` but Apollo has `root: {} 0 keys`

**Desktop**
 - OS: macOS
 - Browser: Chrome 90.0.4430.85
 - Extension version: 3.0.4

**Comment by hwillson** [comment]
@afrankel-sfdo I haven't been able to re-create this. Any change you (or anyone else) could provide a reproduction (tested against 3.0.5)?

**Comment by afrankel-sfdo** [comment]
> @afrankel-sfdo I haven't been able to re-create this. Any chance you (or anyone else) could provide a reproduction (tested against 3.0.5)?

@hwillson is there an easy way to setup a public example apollo client/server app? My test case is private.

**Comment by jeffdesign** [comment]
Hi! First of all, thanks and good job releasing 3.0.5 @hwillson 🎉

Regarding this issue with query variables - I'm able to reproduce it too. I'm using the same client setup I used in https://github.com/apollographql/apollo-client-devtools/issues/462 if it gives any help.

**Comment by axat17** [comment]
I have a similar issue. 

https://github.com/apollographql/apollo-client-devtools/issues/564

**Comment by kosmos** [comment]
I also have a similar issue.

**Comment by benatshippabo** [comment]
@hwillson I think the issue is when we use apollo client v2. See this example here: https://codesandbox.io/s/apollo-client-devtool-variables-94kn5

**Comment by afrankel-sfdo** [comment]
> @hwillson I think the issue is when we use apollo client v2. See this example here: https://codesandbox.io/s/apollo-client-devtool-variables-94kn5

That's also true for us, using v2.

**Comment by afrankel-sfdo** [comment]
@hwillson can you remove `needs-reproducation`, we've determined the issues is related to Apollo v2. What else can we do to help move this along?

**Comment by hwillson** [comment]
Thanks @afrankel-sfdo - the absolute best way to help move this along would be if someone is able to take a quick pass at creating a PR to fix this. We'll definitely get to it soon, but a PR would help expedite things for sure. 🙏

**Comment by afrankel-sfdo** [comment]
@hwillson traced the issue through dev tools, and far as I can tell it's a bug in the client, not passing the variables correctly.

the `query` object here https://github.com/apollographql/apollo-client-devtools/blob/a2e0fbe9f1feec222d78b70e7f2bb494288705b2/src/application/index.tsx#L134 is missing `variables`

<img width="377" alt="Screen Shot 2021-07-28 at 1 47 39 PM" src="https://user-images.githubusercontent.com/31106469/127387335-34c22b45-fb65-4c3b-abd2-09fb450e6b4b.png">

**Comment by afrankel-sfdo** [comment]
@hwillson any insights for the above?

**Comment by msell** [comment]
We also had this same issue, we are currently stuck on Apollo client 2x for the time being.  What I ended up doing is just grabbing the last 2x version of apollo-client-devtools [here](https://github.com/apollographql/apollo-client-devtools/releases/tag/v2.3.5) and building it and installing this old version of the chrome extension.  Once I have this old version the query variables show up again.

**Comment by benatshippabo** [comment]
> We also had this same issue, we are currently stuck on Apollo client 2x for the time being. What I ended up doing is just grabbing the last 2x version of apollo-client-devtools [here](https://github.com/apollographql/apollo-client-devtools/releases/tag/v2.3.5) and building it and installing this old version of the chrome extension. Once I have this old version the query variables show up again.

Yes, that is what my team did as well. Eventually we did migrate to v3 and the migration was pretty painless!
